### ISSUE TITLE
[Bug]: Item duping *High prio*

### ISSUE THREAD

**Issue by Brainiac1830** [Dragon-Crypto-Gaming/DCG-Bug-Reporter#94]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Contact Details

Brainiac#0847

### What happened?

It is possible to dupe items are not unique (resources, potions, $DCAU/$DCAR). Yes... you can technically get minted infinite supply $DCAU/$DCAR or until we reach the token cap I think.

How to do it?

1. Open stash
2. Select an item that is not unique
3. Claim it, but do not accept/refuse the transaction
4.. Close the stash
5. Repeat step 1 to 4 until you're happy with the amount of duping you're about to do.
6. Accept all the transaction.

![image](https://user-images.githubusercontent.com/8630518/206087828-7b8c2114-fae0-4f27-935e-938ada462e0b.png)

### What browsers are you seeing the problem on?

Chrome

### Terms and conditions

- [X] I agree to follow DCG's [terms and conditions](https://dragoncrypto.io/terms-of-use)

**Comment by WesToStakuj** [comment]
I replicated issue on my side for 72units of corrupted bar (ERC1155) in 3tranactions and it shows them all on https://testnet.snowtrace.io/ but in game my inventory/wallet I can see only amount what I had plus 72 what is single transaction. The other 72x2 are not there.

But I did same on TDCAR what is ERC20 and after approving 3transaction the total shows in Grom's Bank is a sum from all transactions and previous balance. It mayor BUG here.
